File: Python/Gato.py
from Mascota import Mascota

class Gato(Mascota):
    def aranhar(self):
        return "Te aranho"

    def comunicarse(self):
        return "miau, miau"

    def presentarse(self):
        return super(Gato, self).presentarse() + " gato"

File: PHP/Gato.php
<?php

require_once 'Mascota.php';

class Gato extends Mascota {

    public function aranhar() {
        return "Te aranho";
    }

    public function comunicarse() {
        return "miau, miau";
    }

    public function presentarse() {
        return parent::presentarse() . "gato";
    }
}
